WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -1499,7 +1499,7 @@ ZaAccountXFormView.myXFormModifier = function(xFormObject, entry) {
         headerItems.push({type:_OUTPUT_, ref:ZaAccount.A_zimbraMailTransport, labelLocation:_LEFT_,label:ZabMsg.attrDesc_mailHost});
     }
 
-    if (ZaItem.hasReadPermission(ZaAccount.A_name, entry)) {
+    if (entry[ZaAccount.A_name]) {
```

Comment:
AjxUtil.isEmpty would be better check

Reply:
@silentsakky I found AjxUtil.isEmailAddress. It seems to be better to check if the value of entry[ZaAccount.A_name] is an email address (string).
Which do you think is better?
- if (!AjxUtil.isEmpty(entry[ZaAccount.A_name])) {
- if (AjxUtil.isEmailAddress(entry[ZaAccount.A_name], false)) {
    // set false to check strictly

Reply:
looks good, we can use it

Reply:
fixed to use AjxUtil.isEmailAddress